low-level/sspl_ll_d

```diff
@@ -722,7 +722,7 @@ def initialize_iem():
     try:
         if os.path.exists(IEM_INIT_FAILED):
             os.remove(IEM_INIT_FAILED)
-        EventMessage.init(component=None, source='S')
+        EventMessage.init(component='sspl', source='S')
```

Comment:
how this helps exactly ?

Reply:
In case of Kafka unavailability, In service_monitor  we call generate_iem function from iem.py file for raisng IEM, but while executing 'EventMessage.send()' it encouters 'kafkaerror' which is not getting handled in
EventMessageError exception, and service monitor thread shutdown due to this unhandled exception, and hence we are not receiving alerts for service, to resolve this issue added exception handling in iem.py file.

Reply:
ok, alright. so it is indeed serving towards actual cause